notebooks/qg/hybridqg_training.ipynb

```diff
@@ -0,0 +1,1032 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multihop Hybrid (Table + text) Question Generation: Training example\n",
+    "In this notebook, we will see how to fine-tune and evaluate a question generation model on HybridQA dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configuration\n",
+    "\n",
+    "We start by setting some parameters to configure the process.  Note that depending on the GPU being used you may need to tune the batch size."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_name_or_path=\"t5-small\"\n",
+    "modality=\"hybrid\"\n",
+    "dataset_name=\"hybrid_qa\"\n",
+    "max_len=200\n",
+    "target_max_len=40\n",
+    "output_dir=\"/dccstor/cssblr/rbhat//models/\"\n",
+    "learning_rate=0.0001\n",
+    "num_train_epochs=2\n",
+    "per_device_train_batch_size=8\n",
+    "per_device_eval_batch_size=32\n",
+    "evaluation_strategy='epoch'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-10-20 01:27:31.845509: E tensorflow/stream_executor/cuda/cuda_blas.cc:2981] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered\n"
+     ]
+    }
+   ],
+   "source": [
+    "from transformers import Seq2SeqTrainingArguments\n",
+    "\n",
+    "training_args = Seq2SeqTrainingArguments(\n",
+    "    output_dir=output_dir,\n",
+    "    overwrite_output_dir=True,\n",
+    "    do_train=True,\n",
+    "    do_eval=True,\n",
+    "    per_device_train_batch_size=per_device_train_batch_size,\n",
+    "    per_device_eval_batch_size=per_device_eval_batch_size,\n",
+    "    num_train_epochs=num_train_epochs,\n",
+    "    evaluation_strategy='epoch',\n",
+    "    learning_rate=learning_rate,\n",
+    "    prediction_loss_only=True,\n",
+    "    remove_unused_columns=False,\n",
+    "    )\n",
+    "training_args.predict_with_generate=True\n",
+    "training_args.remove_unused_columns = False\n",
+    "training_args.prediction_loss_only = False"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "## HybridQA data\n",
+    "Here we load one instance of HybridQA and visualize it. <font color='red'>This part of the code is not needed to train the model </font>"
```

Comment:
I think it would be helpful if you could also show this instance using a table.

Reply:
I am not sure if this nested instance can be easily converted to a table.